website/docs/r/event_trigger.html.markdown

```diff
@@ -0,0 +1,138 @@
+* `config_operation_type` - (Optional) The [authentication operation type](https://docs.mongodb.com/realm/triggers/authentication-triggers/#std-label-authentication-event-operation-types) to listen for. Required for `AUTHENTICATION` type. Possible Values: `LOGIN`, `CREATE`, `DELETE`
+* `config_providers` - (Optional) A list of one or more [authentication provider](https://docs.mongodb.com/realm/authentication/providers/) id values. The trigger will only listen for authentication events produced by these providers. Required for `AUTHENTICATION` type.
+* `config_database` - (Optional) The name of the MongoDB database that contains the watched collection. Required for `DATABASE` type.
+* `config_collection` - (Optional) The name of the MongoDB collection that the trigger watches for change events. The collection must be part of the specified database. Required for `DATABASE` type.
+* `config_service_id` - (Optional) The ID of the MongoDB Service associated with the trigger. Required for `DATABASE` type.
+* `config_match` - (Optional) A [$match](https://docs.mongodb.com/manual/reference/operator/aggregation/match/) expression document that MongoDB Realm includes in the underlying change stream pipeline for the trigger. This is useful when you want to filter change events beyond their operation type. The trigger will only fire if the expression evaluates to true for a given change event. Required for `DATABASE` type.
+* `config_project` - (Optional) A [$project](https://docs.mongodb.com/manual/reference/operator/aggregation/project/) expression document that Realm uses to filter the fields that appear in change event objects. Required for `DATABASE` type.
+* `config_full_document` - (Optional) If true, indicates that `UPDATE` change events should include the most current [majority-committed](https://docs.mongodb.com/manual/reference/read-concern-majority/) version of the modified document in the fullDocument field.
+* `config_schedule` - (Optional) A [cron expression](https://docs.mongodb.com/realm/triggers/cron-expressions/) that defines the trigger schedule.
+* `event_processors` - (Optional) An object where each field name is an event processor ID and each value is an object that configures its corresponding event processor. The following event processors are supported: `AWS_EVENTBRIDGE` For an example configuration object, see [Send Trigger Events to AWS EventBridge](https://docs.mongodb.com/realm/triggers/eventbridge/#std-label-event_processor_example).
+* `event_processors.0.aws_eventbridge.config_account_id` - (Optional) AWS Account ID.
+* `event_processors.0.aws_eventbridge.config_region` - (Optional) Region of AWS Account.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The Terraform's unique identifier used internally for state management.
```

Comment:
slight edit:
 * `id` - Terraform's unique identifier used internally for state management.